test/test_helper.exs

```diff
@@ -14,7 +14,7 @@ alias Ecto.Integration.TestRepo
 Application.put_env(:ecto, TestRepo,
                     adapter: Mongo.Ecto,
                     url: "ecto://localhost:27017/ecto_test",
-                    pool_size: 1)
+                    size: 1)
```

Comment:
I'm using the driver's adapters. Ecto simply translates `pool_size` to `size` before handing options to poolboy.